docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,409 @@
+| parser.content | string | The actual parser definition in the syntax of Fluent Bit. |
+
+### LogParser.status attribute
+
+For details, see the [LogParser specification file](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/apis/telemetry/v1alpha1/logparser_types.go).
+
+| Parameter | Type | Description |
+|-|-|-|
+| conditions | []object | An array of conditions describing the status of the parser.
+| conditions[].lastTransitionTime | []object | An array of conditions describing the status of the parser.
+| conditions[].reason | []object | An array of conditions describing the status of the parser.
+| conditions[].type | enum | The possible transition types are:<br>- Running: The parser is ready and usable.<br>- Pending: The parser is being activated. |
+
+## Log record processing
+
+After a log record has been read, it is preprocessed by some central configuration sections like the `kubernetes` filter. Thus, when a record is ready to be processed by the sections defined in the LogPipeline definition, it will have several attributes available for processing and shipment.
+
+![Flow](./assets/telemetry-logging-flow.drawio.svg)
+
+Learn more about these attributes in the following sections.
+
+### Container log message
+
+In the example, we assume there's a container `myContainer` of Pod `myPod` running in Namespace `myNamespace` logging to `stdout` with the following log message in JSON format:
+
+```json
+{
+  "level": "warn",
+  "message": "This is the actual message",
+  "tenant": "myTenant",
+  "traceID": "123"
+}
+```
+
+### Tail input
+
+The central pipeline tails the log message from a log file managed by the container runtime. The file name contains the Namespace, Pod, and container information that will be available later as part of the [tag](https://docs.fluentbit.io/manual/concepts/key-concepts#tag). The resulting log record available in an internal Fluent Bit representation looks similar to the following example:
+
+```json
+{
+  "time": "2022-05-23T15:04:52.193317532Z",
+  "stream": "stdout",
+  "_p": "F",
+  "log": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"traceID\": \"123\"}
+}
+```
+
+The attributes in the example have the following meaning:
+
+| Attribute | Description |
+|-|-|
+| time | The timestamp generated by the container runtime at the moment the log was written to the log file |
+| stream | The stream to which the application wrote the log, either `stdout` or `stderr` |
+| _p | Indicates if the log message is partial (`P`) or final (`F`). Optional, dependent on container runtime. Because a CRI multiline parser is applied for the tailing phase, all multi-lines on container runtime level are aggregated already and no partial entries should be left. |
```

Comment:
```suggestion
| _p | Indicates if the log message is partial (`P`) or final (`F`). Optional, dependent on container runtime. Because a CRI multiline parser is applied for the tailing phase, all multilines on the container runtime level are aggregated already and no partial entries must be left. |
```